en/flying/tunes.md

```diff
@@ -0,0 +1,156 @@
+# Tune Meanings (Pixhawk Series)
+
+[Pixhawk-series flight controllers](../flight_controller/pixhawk_series.md) use audible tones/tunes to indicate vehicle state and events (e.g. arming success and failure, low battery warnings).
+
+The current set of tones are listed/can be played below.
+
+Startup tone - played on completion of startup, before arming.
+<audio controls>
+  <source src="../../assets/tunes/1_startup_tone.mp3" type="audio/mpeg">
+Your browser does not support the audio element.
+</audio>
+
+Error tune - 
+
+<audio controls>
+  <source src="../../assets/tunes/2_error_tune.mp3" type="audio/mpeg">
+Your browser does not support the audio element.
+</audio>
+
+
+Notify positive tone
+
+<audio controls>
+  <source src="../../assets/tunes/3_notify_positive_tone.mp3" type="audio/mpeg">
```

Comment:
Ditto - when generally is this used. What should user do. Same question for everything ....

Reply:
Should we break this into modes/phases of boot and operational?

For boot see https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/rcS

it describes the logic and what tones are played when.